Classes/Controller/ProfileController.php

```diff
@@ -35,13 +36,14 @@ public function __construct(FeedRepository $feedRepository, TokenRepository $tok
     /**
      * @return void
      */
-    public function showAction()
+    public function showAction(): ResponseInterface
```

Comment:
If it's ok I would leave it as-is and therefore remove the support for TYPO3 10. Since the extension looks like it's feature complete and TYPO3 v12 is coming I would suggest to support only v11 – if OK for you Alex!

I will update therefore the proper constraints.

Reply:
@einpraegsam would love to hear from you

Reply:
Yes, dropping support for 10 is fine also for me